conf.py

```diff
@@ -18,9 +18,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = 'Kinetic'
+version = 'Melodic'
 # The full version, including alpha/beta/rc tags.
-release = 'Kinetic'
+release = 'Melodic'
```

Comment:
will this cause 16.04 machines to no longer work?

Reply:
My understanding is that this is only used in the html <title> tag. It was last changed here:
https://github.com/ros-planning/moveit_tutorials/pull/141